SCons/Node/FS.py

```diff
@@ -937,10 +947,11 @@ def _glob1(self, pattern, ondisk=True, source=False, strings=False):
 
 # Dict that provides a simple backward compatibility
 # layer for the Node attributes 'abspath', 'labspath',
-# 'path', 'tpath' and 'path_elements'.
+# 'relpath', 'path', 'tpath' and 'path_elements'.
 # @see Base.__getattr__ above
 node_bwcomp = {'abspath' : Base.get_abspath,
```

Comment:
Strictly speaking, a new attribute doesn't have any "backward compatibility" concerns, but it probably doesn't hurt to have it here also.

Reply:
Yeah I wasn’t certain about that part

Reply:
Good catch @mwichmann 

@DeeeeLan please remove the backward compat bit.

Reply:
fixed.